modules/ossm-installing-multi-primary-multi-network-kiali.adoc

```diff
@@ -0,0 +1,216 @@
+
+[NOTE]
+====
+In this procedure, `CLUSTER1` is the East cluster and `CLUSTER2` is the West cluster.
+====
+
+You can adapt these instructions for a mesh spanning more than two clusters.
+
+.Prerequisites
+
+* You have installed the latest Kiali Operator on each cluster.
+
+* Istio installed in a multi-cluster configuration on each cluster.
+
+* You have `istioctl` installed on the laptop you can use to run these instructions.
+
+* You are logged in to the {ocp-product-title} web console as a user with the `cluster-admin` role.
+
+* You have configured a metrics store so that Kiali can query metrics from all the clusters. Kiali queries metrics and traces from their respective endpoints.
+
+.Procedure
+
+. Install Kiali on the East cluster:
+
+.. Create a YAML file named `kiali.yaml` that creates a namespace for the Kiali deployment. 
++
+.Example configuration
+[source,yaml,subs="attributes,verbatim"]
+----
+apiVersion: kiali.io/v1alpha1
+kind: Kiali
+metadata:
+  name: kiali
+  namespace: istio-system
+spec:
+  version: default
+  external_services:
+    prometheus:
+      auth:
+        type: bearer
+        use_kiali_token: true
+      thanos_proxy:
+        enabled: true
+      url: https://thanos-querier.openshift-monitoring.svc.cluster.local:9091
+----
++
+[NOTE]
+====
+The endpoint for this example uses OpenShift Monitoring to configure metrics.
+====
+
+.. Apply the YAML file on the East cluster by running the following command:
++
+[source,terminal]
+----
+$ oc --context cluster1 apply -f kiali.yaml
+----
+
+. Ensure that the Kiali CR is ready by running the following command:
++
+[source,terminal]
+----
+$ oc wait --context cluster1 --for=condition=Successful kialis/kiali -n istio-system --timeout=3m
+----
+
+. Display your Kiali Route hostname.
++
+[source,terminal]
+----
+$ oc --context cluster1 get route kiali -n istio-system -o jsonpath='{.spec.host}'
+----
+
+. Create a Kiali CR on the West cluster.
++
+.Example configuration
+[source,yaml,subs="attributes,verbatim"]
+----
+apiVersion: kiali.io/v1alpha1
+kind: Kiali
+metadata:
+  name: kiali
+  namespace: istio-system
+spec:
+  version: default
+  auth:
+    openshift:
+      redirect_uris:
+        # Replace kiali-route-hostname with the hostname from the previous step.
+        - "https://{kiali-route-hostname}/api/auth/callback/cluster2"
+  deployment:
+    remote_cluster_resources_only: true
+----
++
+The Kiali Operator creates the resources necessary for the Kiali server on the East cluster to connect to the West cluster. The Kiali server is not installed on the West cluster.
+
+. Apply the YAML file on the West cluster by running the following command:
++
+[source,terminal]
+----
+$ oc --context cluster2 apply -f kiali-remote.yaml
+----
+
+. Ensure that the Kiali CR is ready by running the following command:
++
+[source,terminal]
+----
+$ oc wait --context cluster1 --for=condition=Successful kialis/kiali -n istio-system --timeout=3m
```

Comment:
```suggestion
$ oc wait --context cluster2 --for=condition=Successful kialis/kiali -n istio-system --timeout=3m
```

Reply:
@rh-tokeefe this is the only thing I see left otherwise the PR is good to go

Reply:
done